gym_solo/envs/solo8v2vanilla.py

```diff
@@ -89,6 +91,7 @@ def reset(self) -> solo_types.obs:
     p.removeBody(self.robot)
     self.robot, _ = self._load_robot()
 
+    # TODO: We need to change this to have the robot always be in home position
```

Comment:
just to be sure, you mean edit the urdf, correct?

Reply:
Yes

Reply:
Do you have any other potentially better ways to do this? @agupta231

Reply:
Nope, anything other than modifying the urdf would be an anti-pattern